spec.html

```diff
@@ -1022,36 +1022,6 @@ <h1>ExecuteModule ( <ins>[ _capability_ ]</ins> ) Concrete Method</h1>
   </emu-clause>
 </emu-clause>
 
-<emu-clause id="sec-toplevelmoduleevaluationjob" aoid="TopLevelModuleEvaluationJob">
```

Comment:
It might be that this was moved into html but I don't know the full context: https://html.spec.whatwg.org/#fetch-the-descendants-of-and-link-a-module-script